test/CMakeLists.txt

```diff
@@ -7,5 +7,8 @@ add_subdirectory(protocol)
 add_subdirectory(hotkey)
 add_subdirectory(storage)
 add_subdirectory(time)
+if(USE_PMEM)
```

Comment:
kind of confused as to what happens when we run cmake without USE_PMEM. would have thought this wouldn't work due to cuckoo needing datapool stuff but maybe i am misunderstanding.

Reply:
By default the pmem implementation of the data pool is not compiled, and instead the shm (`malloc`/`free`) implementation is used as a fallback. What this means is that there's no change whatsoever to the way Pelikan functions, everything will work just the way it used to. Only if it's built with PMEM support explicitly enabled and libpmem installed, the pmem implementation of the datapool abstraction is going to be compiled in, and the `cuckoo_datapool` will be taken into account when starting the cache.

This is implemented in `src/datapool/CMakeLists.txt`.